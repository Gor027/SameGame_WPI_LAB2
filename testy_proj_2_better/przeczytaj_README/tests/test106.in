``Y`` YYYYY` `` Y`Y ` YY `  ``  ` ` `    Y  ```  
``     ```Y`Y
``Y`  Y  YY```Y` `Y Y ` YY Y`Y` ```Y``YY Y  ```Y YYY ``  ` ```Y  `YYYY
 Y   `Y Y
YYY ` `Y     ` `  `  ```YY Y  `  ```Y `Y``Y` 
 YY`YYY``Y`Y``  ` Y Y`` ` YYY  YYY`YY`` `Y  ` `Y`   YY `   Y``Y
 
  Y`Y``YYY`  Y```` YY`Y```Y Y Y Y`Y` `` `Y``Y Y `   ```Y
` Y YYYYY` `Y  Y``Y`Y YY  Y` Y`Y `Y 
YY   Y    `YY `  Y`Y`   YY`` YYYY    `
 `````YYY` Y `YY`` YY YY
  Y`Y  `Y` ` Y`   `YY  ````Y   ``   Y Y``YYY`` Y`Y`
