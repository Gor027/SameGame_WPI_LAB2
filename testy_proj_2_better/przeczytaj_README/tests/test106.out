 `Y   YYYY `        `
 `Y ` Y``` ` `    Y Y Y
 `Y```YYYYY` ``  `Y Y `YY` Y    ` `      Y
`YY`YY```YYYY`Y`Y``YY ``YYYY  ` ``` ` Y  Y  `
`YY`Y`YYYYY```````Y`Y ```YYY```YY``Y``` ``  ` `
`YY`YY`YY``Y`Y````YY``YY``Y``YYYYY``Y``Y`YY````Y`    `
YY```YYYY``YY`YY`Y`YY`YYYY``YY`YY`YYY``Y`Y``Y``Y`YY YY`` ```Y
``Y`Y`Y`Y```YY`Y```YY`Y````YYYY```Y`Y`Y``YYY``YY`Y`Y```Y`Y``Y`YYYY
